store/gcworker/gc_worker.go

```diff
@@ -217,6 +226,32 @@ func createSession(store kv.Storage) session.Session {
 	}
 }
 
+// GetScope gets the status variables scope.
+func (w *GCWorker) GetScope(status string) variable.ScopeFlag {
+	return variable.DefaultStatusVarScopeFlag
+}
+
+// Stats returns the server statistics.
+func (w *GCWorker) Stats(vars *variable.SessionVars) (map[string]interface{}, error) {
+	m := make(map[string]interface{})
+	if v, err := w.loadValueFromSysTable(gcLeaderUUIDKey); err == nil {
+		m[tidbGCLeaderUUID] = v
+	}
+	if v, err := w.loadValueFromSysTable(gcLeaderDescKey); err == nil {
+		m[tidbGCLeaderDesc] = v
```

Comment:
The leader desc usually contains the leader node's hostname or IP address. Since mysql.tidb will be hidden for security concern, can this be exposed directly?

Reply:
We will have to add filtering of `SHOW [GLOBAL] STATUS` to security enhanced mode. The [proposal](https://github.com/pingcap/tidb/pull/22485) is closed for now because it was moved out of this sprint. cc @SunRunAway 

Moving to `SHOW STATUS` is still beneficial though, since filtering from here has a cleaner semantic than patching out results from `SELECT * FROM mysql.tidb`.